pkg/nats/connect.go

```diff
@@ -8,6 +8,7 @@ import (
 
 type Opt = nats.Option
 
+//nolint:gochecknoglobals // cloning functions as variables.
 var (
```

Comment:
Could we store them as consts? Never really tried that but I am wondering.

Reply:
I tried, but it gives error that `nats.RetryOnFailedConnect` is not a const.